build_tools/ci/run_matmul_test.sh

```diff
@@ -584,6 +584,34 @@ run_matmul_test \
     --pipeline "pack-peel" \
     --lhs_rhs_type "i32" \
     --acc_type "i32" \
-    --m "64"  --n "64" --k "160"
+    --m "64"  --n "64" --k "128"
```

Comment:
Is it necessary to change 128 to 160?

Reply:
No both 128 and 160 works. That was an arbitrary choice.